lambda/custom/index.js

```diff
@@ -92,7 +109,8 @@ function handleUserGuess(userGaveUp, handlerInput) {
 
   let speechOutput = '';
   let speechOutputAnalysis = '';
-
+  let speechOutput_APL_1 = '';
+  let speechOutput_APL_2 = '';
```

Comment:
These variable names are unclear. It is not something Alexa is speaking, so it should not be called speechOutput. Also, please stick with camelCase .

Reply:
It is breaking the speechOutput to two pager texts. I can rename it if this is an issue